src/test/java/ma/hibernate/dao/PhoneDaoImplTest.java

```diff
@@ -4,7 +4,6 @@
 import java.util.List;
 import java.util.Map;
 import ma.hibernate.model.Phone;
-import ma.hibernate.util.HibernateUtil;
```

Comment:
I thought we needed to create a HibernateUtil class 🤷‍♀️

Reply:
So did I! But after I have created it - I saw that it is unused, so I deleted it